pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -373,18 +371,14 @@ func (c *ConnectedObjectsQuery) reverseExpandTupleToUserset(
 				Relation:         targetObjectRel,
 				User:             sourceUserRef,
 				ContextualTuples: req.contextualTuples,
-			}, resultChan, req.ingress)
+			}, resultChan, req.ingress, currentQueryCount+1)
```

Comment:
This only adds +1 if we found an object, but it could happen that `storage.ReadStartingWithUser` gave 0 tuples. In that case, we still need to count that read.

Reply:
Good point - we need to make this as an overall count instead of individual object

Reply:
Modified algorithm to have overall count